scripts/delete_resources.sh

```diff
@@ -19,7 +19,7 @@ delete_ec2_instances () {
   else
     echo "EC2 instances matching tag name for the prefix provided $1:"
     EC2_TAG_NAMES=$(aws ec2 describe-instances \
-    --filters "Name=tag:Name,Values=archk3s*" "Name=instance-state-name,Values=running" \
```

Comment:
😱  i din realize i had hard coded something here. so sorry. thanks for catching this :)